src/templates/static/home.jsx

```diff
@@ -158,6 +158,11 @@ const Home = () => {
 
     return (
         <React.Fragment>
+            <div id='msfa_message' className='container gr-padding-30 gr-parent invisible'>
+                <div className='notice-msg center-text gr-parent gr-child'>
+                    <p>{it.L('Financial binary options are only available to professional clients')}</p>
```

Comment:
@kellybinary  Use the current message: `In the EU, financial binary options are only available to professional investors.`